ansible/roles/emqx5/templates/emqx.conf.j2

```diff
@@ -118,7 +118,7 @@ durable_sessions {
 }
 durable_storage {
   messages {
-    backend = {{ emqx_durable_storage_backend | default('builtin') }}
+    backend = {{ emqx_durable_storage_backend | default('builtin_raft') }}
```

Comment:
Hard to say which default is the best here.  I used `builtin_raft` because I assume that most tests will be with EE and/or clusters.

It'll be best to explicitly set this in the test spec.